strawberry_django/filters.py

```diff
@@ -137,7 +138,16 @@ def apply(filters, queryset, pk=UNSET):
     filter_kwargs, filter_methods = build_filter_kwargs(filters)
     queryset = queryset.filter(**filter_kwargs)
     for filter_method in filter_methods:
-        queryset = filter_method(queryset=queryset)
+        argspec = inspect.getfullargspec(filter_method)
+
+        if "info" in getattr(argspec, "args", []) or "info" in getattr(
+            argspec, "kwargs", []
+        ):
+            queryset = filter_method(queryset=queryset, info=info)
+
+        else:
+            queryset = filter_method(queryset=queryset)
```

Comment:
Just worried regarding the performance here.

Maybe add a utility decorated with `@functools.cache` to retrieve the filter method?

Reply:
Added accordingly -- not sure if this is what you meant tho 😅
152cf68ac27ffb46bf54d489aa7c5b299ea8a382

Reply:
Exactly that 👍🏼